dhis-2/dhis-test-web-api/src/test/java/org/hisp/dhis/webapi/controller/deduplication/DeduplicationControllerTest.java

```diff
@@ -177,25 +180,69 @@ void shouldGetNonPaginatedItemsWithPagingSetToFalse() {
   }
 
   @Test
-  void shouldThrowPostPotentialDuplicateWhenMissingDuplicateTeiInPayload() throws Exception {
+  void shouldFailWhenSkipPagingAndPagingAreFalse() {
```

Comment:
shouldFailWhenSkipPagingAndPagingAreFalse and shouldFailWhenSkipPagingAndPagingAreTrue were just deleted. Maybe you need to rebase first